### PR TITLE
added section on legal, politial and ethics and some resources

### PR DESCRIPTION
**Pull request by anagoessens**
I propose to add a section specifically for political, legal and ethical implications of ssi (based on twitter suggestion), I feel the 'philosophy & concepts' section does not fully cover this and would become too broad if they are added. 

Also propose to add KERI resource list and Manning SSI book.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -9,15 +9,19 @@ Self-sovereign identity (SSI) is a term used to describe the digital movement th
...
-- [🛠 Tools & Technologies](#-tools--technologies)
-- [💁‍♂️ Products & Services](#️-products--services)
-- [📡 Networks](#-networks)
-- [📚 Resources](#-resources)
-- [📆 Events](#-events)
-- [💪 Contribute](#-contribute)
-- [🖋️ License](#️-license)
+
+- [Awesome Self-Sovereign Identity](#awesome-self-sovereign-identity)
+  - [Contents](#contents)
```
Comment (TimoGlastra) [inline readme.md]: Is this extra TOC nesting intentional?

Comment (anagoessens) [reply readme.md]: No, probably happened on save

Comment (karimStekelenburg) [reply readme.md]: Assuming you're using the Markdown All in One extension for VSCode, you can fix this by adding ` "markdown.extension.toc.levels": "2..6"` to your settings :)

**Review by karimStekelenburg — APPROVED** [review]
Looks good to me 👏

**Comment by karimStekelenburg** [comment]
@TimoGlastra if you approve as well, feel free to merge!

**Comment by karimStekelenburg** [comment]
Damn, you ninja

**Pull request merged**